tests.py

```diff
@@ -840,6 +847,70 @@ def create_test_pull_job(namespace, quay_host, username, password, concurrency,
     logger.info("Created Job: %s", resp.metadata.name)
 
 
+def parallel_process(user, **kwargs):
```

Comment:
can you write a small description on how this function works and how you are using redis etc

Reply:
Sure. Added the description. Please let me know if you have any further questions on functionality.